build.ps1

```diff
@@ -24,12 +27,13 @@ Write-Output "Using MSBuild from: $msbuild"
 if ($lastexitcode -ne 0) { exit $lastexitcode; }
 
 # Create the stable NuGet package
-& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$env:BUILD_NUMBER"
+echo "Creating package"
+& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix="-$date.git-$hash"
```

Comment:
version number is not exactly the same as the example I gave: https://www.nuget.org/packages/Xamarin.Forms.Background (it is `1.0.0-20191019-1346.git-7150715` when it should be `1.30.0-date20191019-1346.git-7150715`).

Reply:
Sorry for the error. Fixed.

Reply:
also 1.30.0 instead of 1.0.0 (1.3.0 is the last version that Xamarin.Essentials has, try to not hardcode it)

Reply:
I added the latest tag as version.